package.json

```diff
@@ -49,5 +49,8 @@
     "lerna": "lerna",
     "nuke": "rm -r node_modules; for d in packages/*/node_modules; do echo $d; rm -r $d; done",
     "yalc:all": "for d in packages/*/; do echo $d; cd $d; yalc push --replace; cd '../../'; done"
+  },
+  "dependencies": {
+    "boring-avatars": "^1.7.0"
```

Comment:
🛑 This should be a dependency of `clerk-js`, but right now it's in the root-level `package.json`